framework/ChiPhysics/PhysicsMaterial/MultiGroupXS/adjoint_multigroup_xs.h

```diff
@@ -0,0 +1,90 @@
+#ifndef ADJOINT_MULTIGROUP_XS_H
+#define ADJOINT_MULTIGROUP_XS_H
+
+#include "multigroup_xs.h"
+
+
+namespace chi_physics
+{
+
+/**
+ * A class for accessing multi-group cross sections for adjoint problems.
+ */
```

Comment:
We can maybe say a little bit more here. 
The class avoids having to store duplicate data since the only thing that changes is that the transfer- and production matrices are transposed. Therefore we store a reference to the XS this is based off and just reference that data if it is not changed.